packages/checkout/label/index.tsx

```diff
@@ -3,13 +3,13 @@
  */
 import { Fragment } from '@wordpress/element';
 import classNames from 'classnames';
-import type { ReactElement, HTMLAttributes } from 'react';
+import type { ReactElement, HTMLProps } from 'react';
 
-interface LabelProps {
+interface LabelProps extends HTMLProps< HTMLElement > {
```

Comment:
This lets us pass extra props to `Label` which are not defined in the `LabelProps` interface. The specific prop I needed to add this for was `htmlFor` but it makes sense to allow all valid props through.